exonum/fuzz/.gitignore

```diff
@@ -0,0 +1,4 @@
+
+target
+corpus
```

Comment:
why not add these ignores to repo's `.gitignore`? So they all can be found in one place.

```text
./fuzz/target
./fuzz/corpus
./fuzz/artifacts
```

Reply:
The only reason is that it's file was generated automatically by `cargo-fuzz`. Will fix